src/main/java/com/feywild/feywild/entity/ModEntityTypes.java

```diff
@@ -12,10 +12,14 @@
             .sized(0.7f, 1)
             .build(FeywildMod.getInstance().modid + "_bee_knight");
 
-    public static final EntityType<Mandragora> mandragora =  EntityType.Builder.of(Mandragora::new, MobCategory.CREATURE)
+    public static final EntityType<Mandragora> mandragora = EntityType.Builder.of(Mandragora::new, MobCategory.CREATURE)
             .sized(0.7f, 1)
             .build(FeywildMod.getInstance().modid + "_mandragora");
 
+    public static final EntityType<Shroomling> shroomling = EntityType.Builder.of(Shroomling::new, MobCategory.CREATURE)
+            .sized(1, 1)
```

Comment:
Just out of curiosity: This seems pretty big compared to the pictures you sent. Have you verified the hitbox matches the actual entity size?